lib/WebService/Async/Segment/Customer.pm

```diff
@@ -0,0 +1,218 @@
+package WebService::Async::Segment::Customer;
+
+use strict;
+use warnings;
+
+use constant COMMON_FIELDS => qw(context integrations timestamp);
+
+# VERSION
+
+=head1 NAME
+
+WebService::Async::Segment::Customer - represents a customer object with methods to make Segment API calls.
+
+=head1 DESCRIPTION
+
+You can create objects directly or (preferably) indirectly using C<< WebService::Async::Segment::new_customer >>.
+Segment calls B<identify> and B<track> can be triggered on objects of this class.
```

Comment:
Are these methods? If so, they should be `L</identify>` etc. - and if not, why not? Seems logical to have methods for the things that can be called.